### PR TITLE
Add kafka config for how long does kafka query server topic will trigger timeout

### PR DESCRIPTION
**Pull request by zifeihan**
…ger timeout.

Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
Add kafka config for how long does kafka query server topic will trigger timeout

### CODE REVIEW

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaReporterPluginConfig.java [hunk 1]
```diff
@@ -43,6 +43,11 @@
             public static String TOPIC_METER = "skywalking-meters";
 
             public static Map<String, String> PRODUCER_CONFIG = new HashMap<>();
+
+            /**
+             * this config for how long does kafka query server topic will trigger timeout.
+             */
+            public static int GET_TOPIC_TIMEOUT = 5;
```
Comment (wu-sheng) [inline apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaReporterPluginConfig.java:50]: This config should be updated in the agent setup doc.

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaReporterPluginConfig.java [hunk 2]
```diff
@@ -43,6 +43,11 @@
             public static String TOPIC_METER = "skywalking-meters";
 
             public static Map<String, String> PRODUCER_CONFIG = new HashMap<>();
+
+            /**
+             * this config for how long does kafka query server topic will trigger timeout.
```
[the remark this answers is not in the document]
Comment (wu-sheng) [reply apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaReporterPluginConfig.java:48]: Also, missing the unit of this value.

File: apm-sniffer/config/agent.config [hunk 3]
```diff
@@ -103,6 +103,7 @@ plugin.mount=${SW_MOUNT_FOLDERS:plugins,activations}
 
 # Kafka producer configuration
 # plugin.kafka.bootstrap_servers=${SW_KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+# plugin.kafka.get_topic_timeout=${SW_GET_TOPIC_TIMEOUT:5}
```
Comment (wu-sheng) [inline apm-sniffer/config/agent.config:106]: This is not a widely used configuration, please remove it from the default agent.config.

Comment (zifeihan) [reply apm-sniffer/config/agent.config]: done.

**Pull request closed without merging**